src/crs/eqareacylindrical.jl

```diff
@@ -99,13 +99,36 @@ function Base.convert(::Type{EqualAreaCylindrical{latₜₛ,Datum}}, coords::Lat
   e = oftype(l, eccentricity(🌎))
   e² = oftype(l, eccentricity²(🌎))
 
-  k₀ = cos(ϕₜₛ) / sqrt(1 - e² * sin(ϕₜₛ)^2)
   sinϕ = sin(ϕ)
   esinϕ = e * sinϕ
+  k₀ = cos(ϕₜₛ) / sqrt(1 - e² * sin(ϕₜₛ)^2)
   q = (1 - e²) * (sinϕ / (1 - esinϕ^2) - (1 / 2e) * log((1 - esinϕ) / (1 + esinϕ)))
 
   x = a * k₀ * (l - l₀)
   y = a * q / 2k₀
 
   EqualAreaCylindrical{latₜₛ,Datum}(x * u"m", y * u"m")
 end
+
+function Base.convert(::Type{LatLon{Datum}}, coords::EqualAreaCylindrical{latₜₛ,Datum}) where {latₜₛ,Datum}
+  🌎 = ellipsoid(Datum)
+  x = coords.x
+  y = coords.y
+  a = oftype(x, majoraxis(🌎))
+  e = convert(numtype(x), eccentricity(🌎))
+  e² = convert(numtype(x), eccentricity²(🌎))
+  λ₀ = numconvert(numtype(x), deg2rad(longitudeₒ(Datum)))
+  ϕₜₛ = numconvert(numtype(x), deg2rad(latₜₛ))
+
+  ome² = 1 - e²
+  k₀ = cos(ϕₜₛ) / sqrt(1 - e² * sin(ϕₜₛ)^2)
+  # same formula as q, but ϕ = 90°
+  qₚ = ome² * (1 / ome² - (1 / 2e) * log((1 - e) / (1 + e)))
+  
+  λ = λ₀ + x / (a * k₀)
+  q = 2y * k₀ / a
+  β = asin(q / qₚ)
+
+  auth = AuthalicLatLon{Datum}(rad2deg(β) * u"°", rad2deg(λ) * u"°")
+  convert(LatLon{Datum}, auth)
```

Comment:
It can also be written like this:
```suggestion
  λ = λ₀ + x / (a * k₀)
  q = 2y * k₀ / a
  β = asin(q / qₚ)
  ϕ = auth2geod(β, e²)

  LatLon{Datum}(rad2deg(ϕ) * u"°", rad2deg(λ) * u"°")
```
Which would be better?